src/lib.rs

```diff
@@ -423,6 +441,13 @@ impl Capture<Inactive> {
         }
     }
 
+    pub fn nano(self) -> Capture<Inactive> {
```

Comment:
This should allow the caller to enable or disable the precision instead of always enabling it.